google/cloud/bigtable/examples/data_snippets.cc

```diff
@@ -401,6 +402,47 @@ void ReadPrefixList(google::cloud::bigtable::Table table, int argc,
   (std::move(table), prefix_list);
 }
 
+void ReadMultipleRanges(google::cloud::bigtable::Table table, int argc,
+                        char* argv[]) {
+  if (argc < 3) {
+    throw Usage{
+        "read-multiple-ranges: <project-id> <instance-id> <table-id>"
+        " <begin1> <end1> [<begin2> <end2> ...]"};
+  }
+
+  std::vector<std::pair<std::string, std::string>> ranges;
+  for (int i = 1; i != argc - 1; ++i) {
```

Comment:
This doesn't look like it is grouping `<begin> <end>` pairs correctly.

Reply:
Indeed, fixed.